packages/perseus-core/src/data-schema.ts

```diff
@@ -782,7 +782,7 @@ export type PerseusInteractiveGraphWidgetOptions = {
     correct: PerseusGraphType;
     // Shapes (points, chords, etc) displayed on the graph that cannot
     // be moved by the user.
-    lockedFigures?: ReadonlyArray<LockedFigure>;
```

Comment:
Very exciting! The rest of the code for making `lockedFigures` required makes sense to me. Same for `labels`.

Only one question for my own understanding: Is there any possibility of this causing issues for old exercise items that saved this as `null` or `undefined`? Or would that be fixed by the parser?